src/ui/Views/Organisation/About.cshtml

```diff
@@ -9,194 +9,200 @@
   var workflowStatus = Model.GetWorkflowStatus();
 }
 
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <div class="course-basicinfo govuk-!-margin-bottom-9">
-      <h1 class="govuk-heading-xl">
-        <span class="govuk-caption-xl">@Model.TabViewModel.OrganisationName</span>
-        About your organisation
-      </h1>
+<a href="@Url.Action("Courses", "Organisation", new {ucasCode = @Model.TabViewModel.UcasCode})" class="govuk-back-link">Back</a>
+
+<main role="main" class="govuk-main-wrapper" id="main-content">
+  @Html.Partial("~/Views/Shared/_Alerts.cshtml")
```

Comment:
`@await Html.PartialAsync("~/Views/Shared/_Alerts.cshtml")`

Use that instead please

Reply:
Done @defong